source/scripts/model/Game.js

```diff
@@ -25,7 +21,18 @@ export default class Game {
             },
         })
 
-        this.monsters = []
+        this.wave = new MonsterWave({
+            game: this,
+            data: {
+                capacity: 4,
+                monsters: [
+                    MONSTERS.RED_SLIME,
+                    MONSTERS.BLUE_SLIME,
+                ],
+            }
+        })
```

Comment:
I struggled with how to handle that `data` parameter. Eventually, we'll be ready this data (like the capacity and monsters) from a JSON file, so it'd be best to keep it in it's own object.